main.py

```diff
@@ -0,0 +1,85 @@
+import sys
+import time
+
+
+def read(file):
+    try:
+        # file validation
+        with open(file, 'r') as f:
+            l = f.readlines()
```

Comment:
The variable name 'l' should be extended to meaningful way. A variable name should tell what it stores.